.golangci.yml

```
@@ -29,6 +29,8 @@ linters:
               desc: use "jsonrs" instead
             - pkg: github.com/rudderlabs/sonnet
               desc: use "jsonrs" instead
+            - pkg: github.com/aws/aws-sdk-go$
+              desc: use "github.com/aws/aws-sdk-go-v2" instead
```

We need another rule for nested packages too, e.g. `github.com/aws/aws-sdk-go/aws/awserr`
```suggestion
            - pkg: github.com/aws/aws-sdk-go$
              desc: use "aws-sdk-go-v2" instead
            - pkg: github.com/aws/aws-sdk-go/
              desc: use "aws-sdk-go-v2" instead
```